src/OpenTelemetry.OpAmp.Client/.publicApi/PublicAPI.Unshipped.txt

```diff
@@ -1,14 +1,14 @@
-OpenTelemetry.OpAmp.Client.Internal.Listeners.IOpAmpListener
 OpenTelemetry.OpAmp.Client.Internal.Listeners.IOpAmpListener<TMessage>
-OpenTelemetry.OpAmp.Client.Internal.Listeners.IOpAmpListener<TMessage>.HandleMessage(TMessage message) -> void
-OpenTelemetry.OpAmp.Client.Internal.Listeners.IOpAmpMessage
+OpenTelemetry.OpAmp.Client.Internal.Listeners.IOpAmpListener<TMessage>.HandleMessage(TMessage! message) -> void
```

Comment:
It is probably leftover from previous PRs.

Could you please consider separate PR cleaning it up with some short note that Internal infix was removed from the namespaces?

I do not think that we should expose any Internal API

Reply:
Just to make sure I understand correctly, in a new PR, we should both move out of the "Internal" folder and remove from the "Internal" namespace any types we made public in the PR and previously?

Reply:
Raised #3609